pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -63,6 +66,16 @@ type GCECompute interface {
 	DeleteSnapshot(ctx context.Context, snapshotName string) error
 }
 
+// GetProject returns the project that was used to instantiate this GCE client.
+func (cloud *CloudProvider) GetProject() string {
+	return cloud.project
+}
+
+// GetZone returns the zone that was used to instantiate this GCE client.
+func (cloud *CloudProvider) GetZone() string {
```

Comment:
would it make sense to call these `GetDefaultZone` `GetDefaultProject`? I'm not sure if default is the right word but I feel like it needs some sort of qualifier since it's not clear "what zone" are we getting.

Reply:
Sure, done. I called it as you suggested (with "default" now, although, I'm also not sure if that's the best name). If a better name comes into your mind shout it out :)